apps/web/utils/ai/assistant/chat.ts

```diff
@@ -380,6 +383,7 @@ const updateRuleActionsTool = ({
             cc: z.string().nullish(),
             bcc: z.string().nullish(),
             subject: z.string().nullish(),
+            folderName: z.string().nullish(),
```

Comment:
ya folder shouldn't be an option for google. will confuse the ai and can't be used anyway